examples/http-client-data-binding/http_client_data_binding.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - Payload data binding
 
-HTTP client payload data binding feature allows accessing the response payload directly without manipulating the `http:Response` object. The response payload type is inferred from the contextually expected type or the `targetType` argument. `anydata` type or `http:Response` is expected as the expected-type or the `targetType` along with `error` type. When the client payload data binding is expected, the HTTP error responses (4xx, 5xx) are categorized as an `error` (`http:ClientRequestError`, `http:RemoteServerError`) of the client's remote operation. If the expected payload type is not compatible with the response payload type an `http:ClientError` is returned. Use this to access the response payload directly when the response payload type is known beforehand. 
+The `http:Client` payload data-binding allows directly binding the response payload to a given subtype of `anydata`. It does this by mapping a given HTTP content-type to one or more Ballerina types. For instance, `text/plain` is mapped to `string` whereas `application/json` is mapped to `json`, `record`, etc. Client data-binding can be used by simply assigning the resource method’s returned value to the declared variable. If the response is anything other than 2xx, an `error` is returned and no data-binding is performed. If there is no mapping between the given Ballerina type and the response content-type, again an `error` is returned. Use this when the application is only interested in the response payload but not the headers. When the response payload is JSON, `record` type is preferred to `json` type as it provides compile-time validations, better readability and improved tooling support.
```

Comment:
```suggestion
The `http:Client` payload data-binding allows directly binding the response payload to a given subtype of `anydata`. It does this by mapping a given HTTP content-type to one or more Ballerina types. For instance, `text/plain` is mapped to `string`, whereas `application/json` is mapped to `json`, `record`, etc. The client data-binding can be used by simply assigning the resource method’s returned value to the declared variable. If the response is anything other than 2xx, an `error` is returned and no data-binding is performed. If there is no mapping between the given Ballerina type and the response content-type, again an `error` is returned. Use this when the application is only interested in the response payload but not the headers. When the response payload is JSON, the `record` type is preferred to the `json` type as it provides compile-time validations, better readability, and improved tooling support.
```

Reply:
Please refer below comment as we need a few updates WRT this suggestion.